tests/ztimer_xsec/main.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2021 TUBA Freiberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    high level ztimer test application
+ *
+ * @author   Karl Fessel <karl.fessel@ovgu.de>
+ *
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "ztimer.h"
+#include "mutex.h"
+
+#ifndef LOG_LEVEL
+#define LOG_LEVEL LOG_WARNING
+#endif
+
+#include "log.h"
```

Comment:
Since this is a test just use `printfs` IMO, but you can discard this comment if you want.

Reply:
the logs are for manual testing of the timing

Reply:
It doesn't make a difference, but ok as you prefer.